app/src/main/java/org/fossasia/pslab/activity/OscilloscopeActivity.java

```diff
@@ -103,7 +105,11 @@
     Fragment dataAnalysisFragment;
     Fragment xyPlotFragment;
     private final Object lock = new Object();
+    private final Object lock2 = new Object();
+    private final Object lock3 = new Object();
     private Task task;
+    private TaskTwo task2;
```

Comment:
Please change the names to captureTwoTask ,etc.
And I think we can do away with single lock object. What do you think?